elements/pfe-sass/extends/_typography.scss

```
@@ -0,0 +1,71 @@
+%pfe-heading-3xl {
```

```suggestion
%pfe-heading--3xl {
```

BEM suggests modifiers be separated with `--`

---

updated!